examples/Bunch/include/simulation_defines/param/laserConfig.param

```diff
@@ -25,6 +25,21 @@
 
 namespace picongpu
 {
+namespace laser
+{
+/** cell from top where the laser is initialized
+ *
+ * if `laserInitPlaneY == 0` than the absorber are disabled.
+ * if `laserInitPlaneY > absorbercells negative Y` the negative absorber in y
+ * direction is enabled
+ *
+ * valid ranges:
+ *   - laserInitPlaneY == 0
+ *   - absorber cells negative Y < laserInitPlaneY < cells in y direction of the top gpu
+*/
+constexpr uint32_t laserInitPlaneY = 0;
```

Comment:
Maybe this variable should only called `initPlaneY` because we are in the namespace `laser`